packages/office-ui-fabric-react/src/components/Facepile/Facepile.types.ts

```diff
@@ -69,6 +69,9 @@ export interface IFacepileProps extends React.Props<FacepileBase> {
 
   /** Method to access properties on the underlying Persona control */
   getPersonaProps?: (persona: IFacepilePersona) => IPersonaSharedProps;
+
+  /** Set to true to allow for keyboarding to the facepile members when the facepile is placed inside of a focus zone */
+  doNotWrapInFocusZone?: boolean;
```

Comment:
This prop name seems like a code smell, as it exposes some implementation detail.

A more interesting question is why it was added there in the first place, because this seems a bit like misuse of FocusZone. Let me look at the history.